packages/wallets/src/components/WalletListCardTitle/__tests__/WalletListCardTitle.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import WalletListCardTitle from '../WalletListCardTitle';
+
+describe('WalletListCardTitle', () => {
+    it('should render the title with provided currency', () => {
+        render(<WalletListCardTitle title='BTC' />);
```

Comment:
is it just title, or is it currrency-specific title?

Reply:
The title is currency.